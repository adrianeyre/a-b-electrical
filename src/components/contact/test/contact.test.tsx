import { shallow } from 'enzyme';

import Contact from '../contact';

describe('Contact', () => {
	it('Should render correctly', () => {
		const contactComponent = shallow(<Contact />);

		expect(contactComponent).toMatchSnapshot();
	});
});
