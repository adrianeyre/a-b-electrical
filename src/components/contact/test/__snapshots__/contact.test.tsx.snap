// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contact-container"
      >
        <h2
          class="centre-text"
        >
          CONTACT US
        </h2>
        <div
          class="contact-area"
        >
          <ul>
            <li>
              <h3>
                ANDREW BONSER
              </h3>
            </li>
            <li>
              <h4>
                PROPRIETOR
              </h4>
            </li>
            <li>
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-phone "
                data-icon="phone"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M164.9 24.6c-7.7-18.6-28-28.5-47.4-23.2l-88 24C12.1 30.2 0 46 0 64C0 311.4 200.6 512 448 512c18 0 33.8-12.1 38.6-29.5l24-88c5.3-19.4-4.6-39.7-23.2-47.4l-96-40c-16.3-6.8-35.2-2.1-46.3 11.6L304.7 368C234.3 334.7 177.3 277.7 144 207.3L193.3 167c13.7-11.2 18.4-30 11.6-46.3l-40-96z"
                  fill="currentColor"
                />
              </svg>
              <span>
                <a
                  href="tel:07816920771"
                >
                  07816 920771
                </a>
                 / 
                <a
                  href="tel:01623431440"
                >
                  01623 431440
                </a>
              </span>
            </li>
            <li>
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-envelope "
                data-icon="envelope"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48L48 64zM0 176L0 384c0 35.3 28.7 64 64 64l384 0c35.3 0 64-28.7 64-64l0-208L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
                  fill="currentColor"
                />
              </svg>
              <span>
                <a
                  href="mailto:abelectrical29@hotmail.com"
                >
                  abelectrical29@hotmail.com
                </a>
              </span>
            </li>
            <li>
              <a
                href="#"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-facebook "
                  data-icon="facebook"
                  data-prefix="fab"
                  focusable="false"
                  role="img"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M512 256C512 114.6 397.4 0 256 0S0 114.6 0 256C0 376 82.7 476.8 194.2 504.5V334.2H141.4V256h52.8V222.3c0-87.1 39.4-127.5 125-127.5c16.2 0 44.2 3.2 55.7 6.4V172c-6-.6-16.5-1-29.6-1c-42 0-58.2 15.9-58.2 57.2V256h83.6l-14.4 78.2H287V510.1C413.8 494.8 512 386.9 512 256h0z"
                    fill="currentColor"
                  />
                </svg>
                <span>
                  abelectricalnottinghamshire
                </span>
              </a>
            </li>
            <li>
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-house "
                data-icon="house"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M575.8 255.5c0 18-15 32.1-32 32.1l-32 0 .7 160.2c0 2.7-.2 5.4-.5 8.1l0 16.2c0 22.1-17.9 40-40 40l-16 0c-1.1 0-2.2 0-3.3-.1c-1.4 .1-2.8 .1-4.2 .1L416 512l-24 0c-22.1 0-40-17.9-40-40l0-24 0-64c0-17.7-14.3-32-32-32l-64 0c-17.7 0-32 14.3-32 32l0 64 0 24c0 22.1-17.9 40-40 40l-24 0-31.9 0c-1.5 0-3-.1-4.5-.2c-1.2 .1-2.4 .2-3.6 .2l-16 0c-22.1 0-40-17.9-40-40l0-112c0-.9 0-1.9 .1-2.8l0-69.7-32 0c-18 0-32-14-32-32.1c0-9 3-17 10-24L266.4 8c7-7 15-8 22-8s15 2 21 7L564.8 231.5c8 7 12 15 11 24z"
                  fill="currentColor"
                />
              </svg>
              <div
                class="address-container"
              >
                <div>
                  A. B. Electrical & Maintenance
                </div>
                <div>
                  27 Greenbank Drive
                </div>
                <div>
                  Sutton-in-Ashfield
                </div>
                <div>
                  Nottinghamshire
                </div>
                <div>
                  NG17 2DY
                </div>
                <div>
                  United Kingdom
                </div>
              </div>
            </li>
          </ul>
          <iframe
            frameborder="0"
            height="355"
            marginheight="0"
            marginwidth="0"
            scrolling="no"
            src="http://maps.google.co.uk/?ie=UTF8&ll=53.1343522,-1.2652543&spn=0.005694,0.016512&t=m&z=17&output=embed"
            width="100%"
          />
          <br />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contact-container"
    >
      <h2
        class="centre-text"
      >
        CONTACT US
      </h2>
      <div
        class="contact-area"
      >
        <ul>
          <li>
            <h3>
              ANDREW BONSER
            </h3>
          </li>
          <li>
            <h4>
              PROPRIETOR
            </h4>
          </li>
          <li>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-phone "
              data-icon="phone"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M164.9 24.6c-7.7-18.6-28-28.5-47.4-23.2l-88 24C12.1 30.2 0 46 0 64C0 311.4 200.6 512 448 512c18 0 33.8-12.1 38.6-29.5l24-88c5.3-19.4-4.6-39.7-23.2-47.4l-96-40c-16.3-6.8-35.2-2.1-46.3 11.6L304.7 368C234.3 334.7 177.3 277.7 144 207.3L193.3 167c13.7-11.2 18.4-30 11.6-46.3l-40-96z"
                fill="currentColor"
              />
            </svg>
            <span>
              <a
                href="tel:07816920771"
              >
                07816 920771
              </a>
               / 
              <a
                href="tel:01623431440"
              >
                01623 431440
              </a>
            </span>
          </li>
          <li>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-envelope "
              data-icon="envelope"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48L48 64zM0 176L0 384c0 35.3 28.7 64 64 64l384 0c35.3 0 64-28.7 64-64l0-208L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
                fill="currentColor"
              />
            </svg>
            <span>
              <a
                href="mailto:abelectrical29@hotmail.com"
              >
                abelectrical29@hotmail.com
              </a>
            </span>
          </li>
          <li>
            <a
              href="#"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-facebook "
                data-icon="facebook"
                data-prefix="fab"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 256C512 114.6 397.4 0 256 0S0 114.6 0 256C0 376 82.7 476.8 194.2 504.5V334.2H141.4V256h52.8V222.3c0-87.1 39.4-127.5 125-127.5c16.2 0 44.2 3.2 55.7 6.4V172c-6-.6-16.5-1-29.6-1c-42 0-58.2 15.9-58.2 57.2V256h83.6l-14.4 78.2H287V510.1C413.8 494.8 512 386.9 512 256h0z"
                  fill="currentColor"
                />
              </svg>
              <span>
                abelectricalnottinghamshire
              </span>
            </a>
          </li>
          <li>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-house "
              data-icon="house"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M575.8 255.5c0 18-15 32.1-32 32.1l-32 0 .7 160.2c0 2.7-.2 5.4-.5 8.1l0 16.2c0 22.1-17.9 40-40 40l-16 0c-1.1 0-2.2 0-3.3-.1c-1.4 .1-2.8 .1-4.2 .1L416 512l-24 0c-22.1 0-40-17.9-40-40l0-24 0-64c0-17.7-14.3-32-32-32l-64 0c-17.7 0-32 14.3-32 32l0 64 0 24c0 22.1-17.9 40-40 40l-24 0-31.9 0c-1.5 0-3-.1-4.5-.2c-1.2 .1-2.4 .2-3.6 .2l-16 0c-22.1 0-40-17.9-40-40l0-112c0-.9 0-1.9 .1-2.8l0-69.7-32 0c-18 0-32-14-32-32.1c0-9 3-17 10-24L266.4 8c7-7 15-8 22-8s15 2 21 7L564.8 231.5c8 7 12 15 11 24z"
                fill="currentColor"
              />
            </svg>
            <div
              class="address-container"
            >
              <div>
                A. B. Electrical & Maintenance
              </div>
              <div>
                27 Greenbank Drive
              </div>
              <div>
                Sutton-in-Ashfield
              </div>
              <div>
                Nottinghamshire
              </div>
              <div>
                NG17 2DY
              </div>
              <div>
                United Kingdom
              </div>
            </div>
          </li>
        </ul>
        <iframe
          frameborder="0"
          height="355"
          marginheight="0"
          marginwidth="0"
          scrolling="no"
          src="http://maps.google.co.uk/?ie=UTF8&ll=53.1343522,-1.2652543&spn=0.005694,0.016512&t=m&z=17&output=embed"
          width="100%"
        />
        <br />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
